snuba/consumers/consumer.py

```diff
@@ -529,6 +532,7 @@ def process_message(
                 message.value.offset,
                 message.value.partition.index,
                 message.value.timestamp,
+                decoded_headers,
```

Comment:
Is the next step to extract the `decoded_headers` from the `result` (inside of the `MetricsBucketProcessor` and the `GenericMetricsBucketProcessor` and pass it to `BytesInsertBatch`?

Reply:
yeah so like it would be passed in https://github.com/getsentry/snuba/blob/e86fe9a4f12f2641c3b0673ed5ae741c2a8f78a3/snuba/datasets/processors/metrics_bucket_processor.py#L69-L71 

and then we could just do `metadata.headers["sentry_received_timestamp"]` or whatnot in https://github.com/getsentry/snuba/blob/e86fe9a4f12f2641c3b0673ed5ae741c2a8f78a3/snuba/datasets/processors/metrics_bucket_processor.py#L116